examples/stock_example_parallel.jl

```diff
@@ -0,0 +1,97 @@
+#  Copyright 2015, Vincent Leclere, Francois Pacaud and Henri Gerard
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#############################################################################
+# Solving an decision hazard stock problem using multiprocessed dynamic programming :
+# We decide which quantity to store before the randomness happens
+# If too much energy comes, the excess is wasted
```

Comment:
Then present the problem in this setting. The problem is called a stock problem, and the word energy appear : I fear it won't help the user to understand the problem.